classes/user/DAO.php

```diff
@@ -154,13 +154,16 @@ public function getIds(Collector $query): Collection
      */
     public function getByUsername(string $username, bool $allowDisabled = false): ?User
     {
-        $row = DB::table($this->table)
-            ->where('username', '=', $username)
+        $usernameLower = strtolower($username);
+
+        $row = DB::table('users')
+            ->whereRaw("lower(username) = '$usernameLower'")
```

Comment:
You can still use bindings with raw clauses: https://dev.to/stuartcreed/using-sql-bindings-in-laravel-raw-methods-217d